test/es6/rlexe.xml

```diff
@@ -1070,7 +1070,7 @@
   <test>
     <default>
       <files>bug_OS_2553885.js</files>
-      <tags>exclude_win7,BugFix</tags>
+      <tags>exclude_win7,BugFix,exclude_xplat</tags>
```

Comment:
Although UTF8 has no endinannes, No it doesn't miss UTF8 BOM. See below;
```
0000170: 2d0d 0a0d 0aef bbbf 6966 2028 27ed 939b  -.......if ('...
0000180: 272e 6c6f 6361 6c65 436f 6d70 6172 6528  '.localeCompare(
```

`\u1111\u1171\u11B6`
```
0000000: e184 91e1 85b1 e186 b60a
```

So the first unicode character is actually being compared to it's UCS2.

Mentioned one of this disabled tests on PR #1061 discussion.